deck-o-cards.js

```diff
@@ -6,6 +6,20 @@ function deck_o_cards() {
 
   var cards = []; // deck
   var shuffledCards = []; // deck shuffled
+  var topCard = null;
```

Comment:
Rather than creating a variable and setting it to null, you can create AND set at the same time on line 17.